docs/source/share_kedro_viz.md

```diff
@@ -21,116 +21,56 @@ kedro new --starter=spaceflights-pandas
 
 When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
```

Comment:
I think this whole section shouldn't be a part of the intro page anymore. But each tutorial page should start off with this section similar to how the 'Preview Datasets' docs look

This page introduces the feature and different ways to use it  - https://kedro--1757.org.readthedocs.build/projects/kedro-viz/en/1757/preview_datasets.html
This page contains the guide including how to set up spaceflights project - https://kedro--1757.org.readthedocs.build/projects/kedro-viz/en/1757/preview_matplotlib_datasets.html

Reply:
@rashidakanchwala done!